src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -367,6 +378,105 @@ public static double getBlockHeight(final int type, final byte data) {
 				return 1;
 		}
 	}
+
+	/**
+	 * Sends a plugin message using the first player from {@link Bukkit#getOnlinePlayers()}.
+	 *
+	 * The next plugin message to be received through {@code channel} will be assumed to be
+	 * the response.
+	 *
+	 * @param channel the channel for this plugin message
+	 * @param data the data to add to the outgoing message
+	 * @return a completable future for the message of the responding plugin message, if there is one.
+	 * this completable future will complete exceptionally if no players are online.
+	 */
+	public static CompletableFuture<ByteArrayDataInput> sendPluginMessage(String channel, String... data) {
+		return sendPluginMessage(channel, r -> true, data);

```

Comment:
Perhaps use `Predicates::alwaysTrue` here?

Reply:
unfortunately i cant, Predicates holds reference to Guava's impl of Predicate, and java has no comparable field.

Reply:
Guava's impl implements Java's one so it *should* be ok

Reply:
it does not:
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Predicate.html
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Predicates.html

Reply:
It does (20.0+) but since Spigot/MC has updated Guava recently (they were using 17.0 till 1.12), I guess you could leave it that way.